src/components/containers/victory-zoom-container.js

```diff
@@ -131,12 +132,23 @@ export const zoomContainerMixin = (base) => class VictoryZoomContainer extends b
     const childComponents = React.Children.toArray(props.children);
```

Comment:
The fix *must* be in `modifyChildren` or another part of the `VictoryZoomContainer`; it cannot just go in the event handlers (found in `zoom-helpers.js`). This is because if new data is added to a chart, the handlers don't get fired, but we must know whether to show the new data or stay "zoomed in."